repotracker/repotracker_test.go

```diff
@@ -42,12 +42,12 @@ func TestFetchRevisions(t *testing.T) {
 		}
 
 		Convey("Fetching commits from the repository should not return any errors", func() {
-			So(repoTracker.FetchRevisions(ctx, 10), ShouldBeNil)
+			So(repoTracker.FetchRevisions(ctx), ShouldBeNil)
 		})
 
 		Convey("Only get 3 revisions from the given repository if given a "+
 			"limit of 4 commits where only 3 exist", func() {
```

Comment:
To preserve the value of these tests, you'll need to set settings.RepoTracker.NumNewRepoRevisionsToFetch to the values that used to be passed in as a parameter